_P208_Nokia_LCD_5110.ino

```diff
@@ -0,0 +1,290 @@
+        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[1], PSTR(PLUGIN_VALUENAME2_208));
+        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[2], PSTR(PLUGIN_VALUENAME3_208));
+        break;
+      }
+
+    case PLUGIN_WEBFORM_LOAD:{
+        addFormNumericBox(F("Display Contrast(50-100):"), F("plugin_208_contrast"), Settings.TaskDevicePluginConfig[event->TaskIndex][1]);
+
+        addFormPinSelect(F("Backlight PIN"), F("taskdevicepin3"), Settings.TaskDevicePin3[event->TaskIndex]);
+
+        byte choice3 = Settings.TaskDevicePluginConfig[event->TaskIndex][2];
+        int optionValues3[4] = { 0, 1, 2, 3 };
+        String options3[4] = { F("0"), F("90"), F("180"), F("270") };
+        addFormSelector(F("Display Rotation"), F("plugin_208_rotation"), 4, options3, optionValues3, choice3);
+
+        byte choice4 = Settings.TaskDevicePluginConfig[event->TaskIndex][0];
+        String options4[2] = { F("OFF"), F("ON") };
+        int optionValues4[2] = { 0, 1 };
+        addFormSelector(F("Backlight"), F("plugin_208_backlight"), 2, options4, optionValues4, choice4);
+
+        byte choice5 = Settings.TaskDevicePluginConfig[event->TaskIndex][3];
+        byte choice6 = Settings.TaskDevicePluginConfig[event->TaskIndex][4];
+        byte choice7 = Settings.TaskDevicePluginConfig[event->TaskIndex][5];
+        int optionValues5[3] = { 1,2,3 };
+        String options5[3] = { F("normal"), F("large"), F("x-large") };
+        addFormSelector(F("Char.size line-1"), F("plugin_208_charsize_line_1"), 3, options5, optionValues5, choice5);
+        addFormSelector(F("Char.size line-2"), F("plugin_208_charsize_line_2"), 3, options5, optionValues5, choice6);
+        addFormSelector(F("Char.size line-3"), F("plugin_208_charsize_line_3"), 3, options5, optionValues5, choice7);
+
+        char deviceTemplate [lcd_lines][digits_per_line];
+        LoadCustomTaskSettings(event->TaskIndex, (byte*)&deviceTemplate, sizeof(deviceTemplate));
+        for (byte varNr = 0; varNr < lcd_lines; varNr++)
+        {
+          addFormTextBox(String(F("Line ")) + (varNr + 1), String(F("Plugin_208_template")) + (varNr + 1), deviceTemplate[varNr], 80);
+        }
+        success = true;
+        break;
+      }
+
+    case PLUGIN_WEBFORM_SAVE:{
+        Settings.TaskDevicePluginConfig[event->TaskIndex][0] = getFormItemInt(F("plugin_208_backlight"));
+        Settings.TaskDevicePluginConfig[event->TaskIndex][1] = getFormItemInt(F("plugin_208_contrast"));
+        Settings.TaskDevicePluginConfig[event->TaskIndex][2] = getFormItemInt(F("plugin_208_rotation"));
+        Settings.TaskDevicePluginConfig[event->TaskIndex][3] = getFormItemInt(F("plugin_208_charsize_line_1"));
+        Settings.TaskDevicePluginConfig[event->TaskIndex][4] = getFormItemInt(F("plugin_208_charsize_line_2"));
+        Settings.TaskDevicePluginConfig[event->TaskIndex][5] = getFormItemInt(F("plugin_208_charsize_line_3"));
+
+        char deviceTemplate[lcd_lines][digits_per_line];

```

Comment:
You're allocating 300 bytes on the stack, which is only 4k in size.
So depending on when it is called, you may cause a stack overflow.
Make sure the highest recorded stack usage is low enough so it is unlikely you will run into a stack overflow.